### ISSUE TITLE
I s there a type definition file for socket ? 

### ISSUE THREAD

**Issue by RobinSrimal** [socketsupply/create-socket-app#66]

**Comment by heapwolf** [comment]
Yes, it's distributed as a part of the npm module, you can also get it [here](https://github.com/socketsupply/socket/blob/master/api/index.d.ts). Is there a way you think this could be improved?

**Comment by RobinSrimal** [comment]
My understanding is that the npm package does not work?  I cloned the repo and ran install.sh instead. 

I stumbled over this when using create socket app with react-ts. It would be good if that would just work out of the box without errors. I tried copying the type definition file into my repo but got errors in the type definition file itself. Happy to share those if needed but I will stick with vanilla for the time being.
